### PR TITLE
introduce configuration

### PR DESCRIPTION
**Pull request by vitalikdanchenko**

### CODE REVIEW

File: lib/talkable/configuration.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module Talkable
+  class Configuration
+    attr_accessor :site_slug
+    attr_accessor :api_key
+    attr_accessor :server
+    attr_accessor :js_integration_library
```
Comment (Tonkonozhenko) [inline lib/talkable/configuration.rb:6]: I think we can merge this in one row

File: lib/talkable/configuration.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    def initialize
+      self.site_slug  = ENV["TALKABLE_SITE_SLUG"]
+      self.api_key    = ENV["TALKABLE_API_KEY"]
+      self.server     = "https://talkable.com"
+    end
+
+    def apply(config)
+      config.each do |key, value|
+        send("#{key}=", value) if respond_to?("#{key}=")
```
Comment (Tonkonozhenko) [inline lib/talkable/configuration.rb]: I think we need raise error if method missing

Comment (Tonkonozhenko) [inline lib/talkable/configuration.rb]: And use `public_send` here

**Comment by Tonkonozhenko** [comment]
Do we need to automate site creation?

File: lib/talkable/configuration.rb (hunk 2 above)
Comment (vitalikdanchenko) [inline lib/talkable/configuration.rb]: `respond_to?` checks only public methods if you don't explicitly specify to look among private methods as well. But anyway `public_send` looks safer.

**Comment by vitalikdanchenko** [comment]
definitely not now

**Comment by Tonkonozhenko** [comment]
LGTM

**Pull request merged**